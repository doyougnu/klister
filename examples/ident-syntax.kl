#lang kernel

(import "prim-syntax.kl")

(example
    (ident-syntax 'one 'two))
