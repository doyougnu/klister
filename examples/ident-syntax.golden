Type mismatch at ident-syntax.kl:6.19-6.23. Expected String but got Syntax
